images/capi/Makefile

```
@@ -318,7 +318,7 @@ PACKER_POWERVS_NODE_FLAGS := $(foreach f,$(abspath $(COMMON_POWERVS_VAR_FILES)),
 CENTOS_VERSIONS			:=	centos-7
 FLATCAR_VERSIONS		:=	flatcar
 PHOTON_VERSIONS			:=	photon-3 photon-4 photon-5
-RHEL_VERSIONS			:=	rhel-7 rhel-8
+RHEL_VERSIONS			:=	rhel-7 rhel-8 rhel-9
```

Undoing this change should fix the tests
```suggestion
RHEL_VERSIONS			:=	rhel-7 rhel-8
```